lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -633,6 +633,19 @@ radius_server:
     config_set: '<state> radius-server host <ip> key <key>'
     default_value: null
 
+radius_server_group:
+  groups:
+    config_get: "show running-config radius all"
+    config_get_token: '/^aaa group server radius (.*) $/'
+    config_set: '<state> aaa group server radius <name>'
+    default_value: ''
+
+  hosts:
+    config_get: "show running-config radius all"
+    config_get_token: ['/^aaa group server radius %s $/i', '/server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/']
+    config_set: ['aaa group server radius <group>', '<state> server <ip>']
+    default_value: ''
```

Comment:
@mikewiebe I was wondering about using _template with this since there are other commands under the group? I removed an earlier comment because I'm not sure if it would actually work in this case.